### PR TITLE
Fix merge-vs-deps task

### PR DESCRIPTION
**Pull request by genlu**

### CODE REVIEW

**Comment by genlu** [comment]
@JoeRobich @allisonchou

**Review by allisonchou — APPROVED** [review]

**Review by JoeRobich — APPROVED** [review]

File: scripts/merge-vs-deps.ps1 [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-    [string]$userName,
-    [string]$userEmail,
-    [string]$accessToken = "")
+Param([string]$accessToken = "")
 
-git config user.name $userName
-git config user.email $userEmail   
+# name and email are only used for merge commit, it doesn't really matter what we put in there.    
+git config user.name "RoslynValidation"
+git config user.email "validation@roslyn.net"
```
Comment (jasonmalinowski) [inline scripts/merge-vs-deps.ps1:6]: Let's use something ending in `@microsoft.com` or before we discover that this does matter and the poor owner of roslyn.net is surprised. :smile:

Comment (jasonmalinowski) [reply scripts/merge-vs-deps.ps1:6]: (We do own roslyn.io if you want to be hip and use that.)

Comment (RikkiGibson) [reply scripts/merge-vs-deps.ps1:6]: @dotnet-bot has dotnet-bot@dotnetfoundation.org :)

**Pull request merged**